tests/schema_change_test.cc

```diff
@@ -715,3 +715,24 @@ SEASTAR_TEST_CASE(test_schema_digest_does_not_change_with_functions) {
             e.execute_cql("create aggregate my_agg(int) sfunc my_add stype int finalfunc twice;").get();
         });
 }
+
+SEASTAR_TEST_CASE(test_schema_digest_does_not_change_with_cdc_options) {
+    std::vector<utils::UUID> expected_digests{
+        utils::UUID("a1f07f31-59d6-372a-8c94-7ea467354b39"),
+        utils::UUID("524d418d-a2e2-3fc3-bf45-5fb79b33c7e4"),
+        utils::UUID("524d418d-a2e2-3fc3-bf45-5fb79b33c7e4"),
+        utils::UUID("018fccba-8050-3bb9-a0a5-2b3c5f0371fe"),
+        utils::UUID("018fccba-8050-3bb9-a0a5-2b3c5f0371fe"),
+        utils::UUID("58f4254e-cc3b-3d56-8a45-167f9a3ea423"),
+        utils::UUID("48fda4f8-d7b5-3e59-a47a-7397989a9bf8"),
+        utils::UUID("8049bcfe-eb01-3a59-af33-16cef8a34b45"),
+        utils::UUID("2195a821-b2b8-3cb8-a179-2f5042e90841")
+    };
+    return test_schema_digest_does_not_change_with_disabled_features(
+        "./tests/sstables/schema_digest_test_cdc_options",
+        std::set<sstring>{},
```

Comment:
How were those sstables created? Should be documented here.

Reply:
They were generated the same ways as sstables for other test_schema_digest_does_not_change_with_disabled_features tests. There's a regenerate flag that you set to true to generate them.

Reply:
@haaawk and where was *that* documented?
There is some general "NOTICE" about "Regenerating data for this test may be necessary when a system ta
ble is added." but it is completely mysterious what it means, and what happens when you regenerate - I would be worried that if I regenerate the tables I would just make the existing bug permanent by committing wrong "expected" values. There definitely needs to be some more precise documentation on how these tests work, and how to properly "generate" them.

Reply:
That's a question for the creator of the test @nyh. I can't speed for their intentions. What I did was to look at how it was done before, talked to Sarna how he did that last time and followed the same route. Of course you can persist a bug by committing wrong expected values if you're not careful.